config/installers/rolebasedcreds/kustomization.yaml

```diff
@@ -4,12 +4,14 @@ kind: Kustomization
 # Adds namespace to all resources.
```

Comment:
Under this organization, we have the following structure:

```
config
    default
    installers
        rolebased
            namespaced
    ...
```

I think it is better to have:

```
config
    default # later we can remove this or move it into installers
    installers # (all rolebased)
        clusterscope
        namespace
    ...
```
IMO having `namespaced` be a child of `cluster-scope rolebased` is bad because it conveys the incorrect idea that `namespaced` depends on `clusterscope`, which it doesn't. Additionally it is good idea to avoid unnecessary nesting.

Any reason why `namespaced` needs to be a child of `rolebased`?

This might be better as separate comment, but maybe we should structure release directory same way? `namespace` and `clusterspace` are siblings, one not dependent on the other.

Reply:
I would love to redesign the structure of these folders, and completely agree it is a little nonsensical at the moment. However I am forced to make silly children folders because Kustomize restricts any resources to be within a child folder of the Kustomization target. Therefore if I want to share resources between two separate targets I have to put one of them into a child folder of the other. If you have better naming I'd happily accept those, though.